<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointToPlane</class>
 <widget class="QWidget" name="PointToPlane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pivot A:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pivot B:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Normal B:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Vec3Setter" name="widgetPivotA" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="Vec3Setter" name="widgetPivotB" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="Vec3Setter" name="widgetNormalB" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vec3Setter</class>
   <extends>QWidget</extends>
   <header>vec3setter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
